verl/trainer/config/_generated_ppo_trainer.yaml

```diff
@@ -385,6 +354,62 @@ algorithm:
   pf_ppo:
     reweight_method: pow
     weight_pow: 2.0
+trainer:
+  balance_batch: true
+  total_epochs: 30
+  total_training_steps: null
+  controller_nsight_options:
```

Comment:
Nsight Systems needs to clarify controller and worker options. We may consider where is the best place to maintain them in your new API system. How about in `global_profiler.nsys.*`